src/elements/ComicScroll.tsx

```diff
@@ -0,0 +1,27 @@
+import ComicCard from "../atoms/ComicCard";
+import { IComicData } from "../models/comicData";
+function ComicScroll({
+  heading,
+  comics,
+}: {
+  heading: string;
+  comics: IComicData[];
+}) {
+  return (
+    <>
+      <h4 className="text-medium font-medium">{heading}</h4>
+      <div className="major-cities-scroll bg-white overflow-auto whitespace-nowrap mb-6">
+        {comics.map((comic) => (
```

Comment:
Its recommended to add a key when mapping items.

Reply:
Oh yes, thank you!